server/views/pages/confirm-instructions.njk

```diff
@@ -2,13 +2,25 @@
 {%- from "moj/components/sub-navigation/macro.njk" import mojSubNavigation -%}
 {% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
 {%- from "govuk/components/fieldset/macro.njk" import govukFieldset -%}
-{% from "../components/instructions/macro.njk" import instructions %}
-{% from "../components/pop/macro.njk" import pop %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
 {%- from "govuk/components/input/macro.njk" import govukInput -%}
 {%- from "govuk/components/button/macro.njk" import govukButton -%}
 {% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "../components/instructions/macro.njk" import instructions %}
+{% from "../components/pop/macro.njk" import pop %}
 
 
+{% set emailCheckbox %}
+{{ govukCheckboxes({
+                          name: 'emailCopy',
```

Comment:
How come there isn't a reference to this in the code? How does it get used?

Reply:
It's just sent in a form post, trying to keep it simple. It doesn't serve any purpose in this application - the processing is in hmpps-workload

Reply:
Ah I get it now, naively I thought we were sending the form directly to the sever but I see now that in the service we are just picking every field off it and sending it in a very slightly different format. So in fact this is never getting sent. I think there is probably a missing test here that checks what gets posted when post is clicked.